truffle/src/main/java/org/jruby/truffle/pack/nodes/PackNode.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2015 Oracle and/or its affiliates. All rights reserved. This
+ * code is released under a tri EPL/GPL/LGPL license. You can use it,
+ * redistribute it and/or modify it under the terms of the:
+ *
+ * Eclipse Public License version 1.0
+ * GNU General Public License version 2
+ * GNU Lesser General Public License version 2.1
+ */
+package org.jruby.truffle.pack.nodes;
+
+import com.oracle.truffle.api.CompilerAsserts;
+import com.oracle.truffle.api.CompilerDirectives;
+import com.oracle.truffle.api.dsl.ImportStatic;
+import com.oracle.truffle.api.dsl.TypeSystemReference;
+import com.oracle.truffle.api.frame.FrameSlotTypeException;
+import com.oracle.truffle.api.frame.VirtualFrame;
+import com.oracle.truffle.api.nodes.Node;
+import com.oracle.truffle.api.utilities.ConditionProfile;
+import org.jruby.truffle.pack.runtime.PackFrame;
+import org.jruby.truffle.pack.runtime.exceptions.TooFewArgumentsException;
+import org.jruby.util.ByteList;
+
+import java.util.Arrays;
+
+/**
+ * The root of the pack nodes.
+ * <p>
```

Comment:
The `<p>`? That's a common notation in JavaDoc. I thought it was the correct way to do things?